test/mobile/nnc/CMakeLists.txt

```diff
@@ -15,8 +15,16 @@ target_link_libraries(test_mobile_nnc PRIVATE torch gtest)
 target_include_directories(test_mobile_nnc PRIVATE ${ATen_CPU_INCLUDE})
 target_compile_definitions(test_mobile_nnc PRIVATE USE_GTEST)
 
+add_executable(test_aot_model_compiler
```

Comment:
nit: shall we just name it "aot_model_compiler"? The test prefixed executables should be doing testing, while this binary itself doesn't test anything, right?

It just so happens that we build the aot compiler binary within the unit test CMakeLists file because the default libtorch CI build job doesn't build binaries, but we might change that behavior sometime in the future. So it's better not to have the rest of the test script depends on this odd fact (i.e. just calling the binary as is, rather than prefixing it with "test").

Reply:
That is right, the binary itself doesn't do the testing but it's used for testing. 
Named it with test prefix to differentiate it from the actual binary https://github.com/pytorch/pytorch/blob/master/binaries/CMakeLists.txt#L112 until we change the behavior.